build/template-nuget-pack.yaml

```diff
@@ -5,10 +5,9 @@ parameters:
   NoBuild: 'true'
 
 steps:
-- task: VSBuild@1
+- task: DotNetCoreCLI@2
```

Comment:
Use dotnet, rather than VSBuild as we are working with preview versions of dotnet core (and anyway we don't need the VS task)